libs/sdk-ui-web-components/api/sdk-ui-web-components.api.md

```diff
@@ -0,0 +1,21 @@
+## API Report File for "@gooddata/sdk-ui-web-components"
```

Comment:
I have 3 entry points for the repo - index for the main + one for each backend. This way we can load the absolute minimum in the first chunk and load the rest lazily. And user can still load tiger/bear backend in case they want custom auth setup. API extractor only works with a single entry point. I don't see it as issue, as the other two entry points are described in tiger and backend packages respectively.

Reply:
I guess that is ok for now :) please add this package to common/scripts/build-docs.js to the libsToBeRemoved array so that it is not made part of the API reference.